ff/serving/provider/online.go

```diff
@@ -114,6 +147,34 @@ func NewRedisOnlineStore(options *RedisConfig) *redisOnlineStore {
 	}
 }
 
+func NewCassandraOnlineStore(options *CassandraConfig) (*cassandraOnlineStore, error) {
+
+	//Create cluster and session
+	cassandraCluster := gocql.NewCluster(options.Addr)
+	var newSession *gocql.Session
+	newSession, err := cassandraCluster.CreateSession()
+	if err != nil {
+		panic(err)
```

Comment:
format of NewCassandraonlinestore returns an error as a second return variable, so you can return nil, err here